config/initializers/course_study_modes.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+COURSE_STUDY_ENUMS = {
+  part_time: "part_time",
+  full_time: "full_time",
+  part_time_or_full_time: "part_time_or_full_time",
+}.freeze
+
+COURSE_STUDY_MODES = {
+  COURSE_STUDY_ENUMS[:part_time] => 0,
+  COURSE_STUDY_ENUMS[:full_time] => 1,
+  COURSE_STUDY_ENUMS[:part_time_or_full_time] => 2,
```

Comment:
I'm not sure on this - are we ever going to let a user select `part_time_or_full_time`? I know that publish courses can be `part_time_or_full_time`, but I _believe_ that in this case we are going to force the user to choose between the two distinct options. @gpeng - is this right?

I don't think there's a `full_time_or_part_time` option in DTTP either.

Reply:
@AbigailMcP
We are letting the user choose when it comes from publish courses, but it will be imported into Trainee before the choice it made I'm guessing?
We can remove it from the enum list if not. In the view file, we only show full and part time, not both.

Reply:
I _think_ it will be imported onto the course, but not saved on trainee https://trello.com/c/XBDN1FnL/2328-l-course-study-mode-question-publish-trainees

Let's remove it from the list in this PR - we can always add it back if / when needed 👍

Reply:
Remove 👍